azure/opsverse-aks/backend.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "<= 4.4.0"
+    }
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "<->"         # Resource group of pre-created storage account for storing Terraform state.
```

Comment:
If this is a placeholder for the actual value, we should add a note in the README asking the customer to substitute this with appropriate values. README should be a single source of information. All these details must be there in the README.

Reply:
In the prerequisites section, I have specified the need for a storage account and a storage container. Since all Azure resources must be associated with a resource group, should I explicitly state that a resource group should be created before these resources.